crowbar_framework/app/models/service_object.rb

```diff
@@ -1529,8 +1536,6 @@ def run_remote_chef_client(node, command, logfile_name)
       ssh = ["sudo", "-u", "root", "--", "ssh", "-o", "TCPKeepAlive=no", "-o", "ServerAliveInterval=15", "root@#{node}"]
       ssh_cmd = ssh.dup << command
 
-      # check if there are currently other chef-client runs on the node
-      wait_for_chef_clients(node)
```

Comment:
you are probably right. I could test it and see if it fails, but I would rather keep it here as well for being on the safe side. will update the PR